sdk/storage/azure-storage-file-share/swagger/README.md

```diff
@@ -15,7 +15,7 @@ autorest --use=@microsoft.azure/autorest.java@3.0.4 --use=jianghaolu/autorest.mo
 
 ### Code generation settings
 ``` yaml
-input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/storage-dataplane-preview/specification/storage/data-plane/Microsoft.FileStorage/preview/2020-02-10/file.json
+input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/467d3ca51bb715fea928251f1003515aee67919a/specification/storage/data-plane/Microsoft.FileStorage/preview/2019-12-12/file.json
```

Comment:
Do we want to actually merge this?

Reply:
No, Sean needs to first merge the swagger PR and then I can update it and regenerate before I merge this